core/server/api/canary/identities.js

```diff
@@ -0,0 +1,36 @@
+const settings = require('../../services/settings/cache');
+const urlUtils = require('../../lib/url-utils');
+const jwt = require('jsonwebtoken');
+const jose = require('node-jose');
+const issuer = urlUtils.urlFor('admin', true);
+
+const dangerousPrivateKey = settings.get('ghost_private_key');
+const keyStore = jose.JWK.createKeyStore();
+const keyStoreReady = keyStore.add(dangerousPrivateKey, 'pem');
+
+const getKeyID = async () => {
+    const key = await keyStoreReady;
+    return key.kid;
+};
+
+const sign = async (claims, options) => {
+    const kid = await getKeyID();
+    return jwt.sign(claims, dangerousPrivateKey, Object.assign({
+        issuer,
+        expiresIn: '30m',
+        audience: '@TODO@',
```

Comment:
@allouis should we set the audience bit within this PR? Not sure if this should be treated similar way as admin tokens, where audience is and Admin API URL e.g.: http://ghost.local/ghost/api/v3/admin/? Don't know if this will be even needed on the verifier's side :thinking:

Reply:
I've removed it for now